src/client/helpers/helpers.js

```diff
@@ -10,6 +10,100 @@ const binByKey = (key, xs) =>
     return binnedArray;
   }, {});
 
+
+function showTaggedMemory(memoryToShow) {
+  $('.memory').each(function () {
+    if ($(this).hasClass(memoryToShow)) {
+      $(this).show();
+    }
+  });
+}
+
+function showMemoriesByActiveTags() {
+  $('.memory').hide();
+  $('.tagLabel').each(function () {
+    if ($(this).is(':visible')) {
+      const tag = $(this).text();
+      showTaggedMemory(tag);
+    }
+  });
+}
+
+function hideOtherTags(tagToKeep) {
+  $('.tagLabel').each(function () {
+    const tagLabel = $(this).text().replace(/\s+/g, '');
+    if (tagLabel !== tagToKeep) {
+      $(this).parent().hide();
+    }
+  });
+}
+
+function hideOtherMemories(memoryTagToKeep) {
+  $('.memory').each(function () {
+    if (!$(this).hasClass(memoryTagToKeep)) {
+      $(this).hide();
+    }
+  });
+}
+
+function tagSorting() {
+  $('.tagLabel').on('click', function () {
+    const clickedTag = $(this).text().replace(/\s+/g, '');
+    hideOtherTags(clickedTag);
+    hideOtherMemories(clickedTag);
+  });
+}
+
+function clearButton() {
+  $('.clear-tags').on('click', () => {
+    $('.memory').show();
+    $('.tag-container').show();
+  });
+}
+
+function removingTags() {
+  $('.filter-tags').on('click', function () {
+    $(this).parent().hide();
+    showMemoriesByActiveTags();
+  });
+}
+
+function closeTagMenu() {
+  $('.close-tags').on('click', () => {
+    $('.underline').removeClass('active');
+    setTimeout(() => {
+      $('.shuffle-memories').fadeIn(1000);
```

Comment:
You have variables like, 1000, and 500 scattered around everywhere.

Make a variable?